storage/innobase/dict/dict0crea.cc

```diff
@@ -2541,6 +2542,109 @@ dict_create_or_check_sys_tablespace(void)
 	return(err);
 }
 
+/****************************************************************//**
+Creates the sys_columns_added system tables inside InnoDB at server 
+bootstrap or server start if they are not found or are not of the 
+right form.
+@return DB_SUCCESS or error code */
+dberr_t
+dict_create_or_check_sys_columns_added(void)
+/*=====================================*/
```

Comment:
Write `()` instead of `(void)`, and remove the `/*======*/` and `/******/` comments.